core/commonTest/src/kotlinx/serialization/UmbrellaTypes.kt

```diff
@@ -68,8 +68,8 @@ data class ArraysUmbrella(
 }
 
 val umbrellaInstance = TypesUmbrella(
-    Unit, true, 10, 20, 30, 40, 50.1f, 60.1, 'A', "Str0", Attitude.POSITIVE, IntData(70),
-    null, null, 11, 21, 31, 41, 51.1f, 61.1, 'B', "Str1", Attitude.NEUTRAL, null,
+    Unit, true, 10, 20, 30, 40, 50.5f, 60.5, 'A', "Str0", Attitude.POSITIVE, IntData(70),
```

Comment:
I'd rather avoid changing test data if WASM has some problems with float numbers. You may add some `if`s around here if you need to.

Reply:
It is quite random numbers. The new ones is not better or worse than old ones :)
You think that it is nice to have here?

Reply:
If they're random, why change them?

Reply:
Agggh...because this data is not suitable to wasm due to lack of wasm stdlib implementation. If i will add another dataset only for wasm how can it make things better?

Reply:
If the "random" values break something we got lucky that they detect a bug. They shouldn't be changed. Rather the bug fixed, or the test "ignored" if that is best.

Reply:
So old ones ARE better, given that they catch bugs, aren't they? :) 

Good idea with two tests!